plugin/cmd/kube-scheduler/app/configurator.go

```diff
@@ -186,3 +191,100 @@ func (sc schedulerConfigurator) Create() (*scheduler.Config, error) {
 
 	return sc.CreateFromConfig(*policy)
 }
+
+func (sc *schedulerConfigurator) SetupPolicyConfigMapEventHandlers(client clientset.Interface, informerFactory informers.SharedInformerFactory) {
+	// selector targets only the scheduler's policy ConfigMap.
+	selector := cache.NewListWatchFromClient(client.CoreV1().RESTClient(), "configmaps", sc.policyConfigMapNamespace, fields.OneTermEqualSelector(api.ObjectNameField, string(sc.policyConfigMap)))
```

Comment:
I tried to go over the advantages and disadvantages of using a file vs. a ConfigMap in [this design doc](https://docs.google.com/document/d/19AKH6V6ejOeIvyGtIPNvRMR4Yi_X8U3Q1zz2fgTNhvM/edit). Given that we still support legacy config files as well, we thought using a ConfigMap would be simpler to implement and won't sacrifice flexibility. That part is already merged as a different PR, but if there are concerns which are not addressed, I will be happy to have a meeting to discuss the options.